src/sage/schemes/berkovich/berkovich_space.py

```diff
@@ -315,59 +314,59 @@ class Berkovich_Cp_Affine(Berkovich_Cp):
 
     EXAMPLES::
 
-        sage: B = Berkovich_Cp_Affine(3); B
+        sage: B = Berkovich_Cp_Affine(3); B                                             # optional - sage.rings.padics
         Affine Berkovich line over Cp(3) of precision 20
 
     We can create elements::
 
-        sage: B(-2)
+        sage: B(-2)                                                                     # optional - sage.rings.padics
         Type I point centered at 1 + 2*3 + 2*3^2 + 2*3^3 + 2*3^4 + 2*3^5
         + 2*3^6 + 2*3^7 + 2*3^8 + 2*3^9 + 2*3^10 + 2*3^11 + 2*3^12 + 2*3^13
         + 2*3^14 + 2*3^15 + 2*3^16 + 2*3^17 + 2*3^18 + 2*3^19 + O(3^20)
 
     ::
 
-        sage: B(1, 2)
+        sage: B(1, 2)                                                                   # optional - sage.rings.padics
         Type III point centered at 1 + O(3^20) of radius 2.00000000000000
 
     For details on element creation, see the documentation
     of :class:`Berkovich_Element_Cp_Affine`. Initializing by
     passing in `\QQ_p` looks the same::
 
-        sage: B = Berkovich_Cp_Affine(Qp(3)); B
+        sage: B = Berkovich_Cp_Affine(Qp(3)); B                                         # optional - sage.rings.padics
         Affine Berkovich line over Cp(3) of precision 20
 
     However, this method allows for more control over behind-the-scenes conversion::
 
-        sage: B = Berkovich_Cp_Affine(Qp(3, 1)); B
+        sage: B = Berkovich_Cp_Affine(Qp(3, 1)); B                                      # optional - sage.rings.padics
         Affine Berkovich line over Cp(3) of precision 1
 
-        sage: B(1/2)
+        sage: B(1/2)                                                                    # optional - sage.rings.padics
         Type I point centered at 2 + O(3)
 
     Note that this point has very low precision, as ``B`` was initialized
     with a p-adic field of capped-relative precision one. For high precision,
     pass in a high precision p-adic field::
 
-        sage: B = Berkovich_Cp_Affine(Qp(3, 1000)); B
+        sage: B = Berkovich_Cp_Affine(Qp(3, 1000)); B                                   # optional - sage.rings.padics
         Affine Berkovich line over Cp(3) of precision 1000
 
     Points of Berkovich space can be created from points of
     extensions of `\QQ_p`::
 
-        sage: B = Berkovich_Cp_Affine(3)
-        sage: A.<a> = Qp(3).extension(x^3 - 3)
-        sage: B(a)
+        sage: B = Berkovich_Cp_Affine(3)                                                # optional - sage.rings.padics
+        sage: A.<a> = Qp(3).extension(x^3 - 3)                                          # optional - sage.rings.number_field
```

Comment:
Extensions of p-adic fields seems to still belong to `sage.rings.padics`.

Reply:
Thanks, fixed.